merkato/merkato.py

```diff
@@ -211,6 +334,7 @@ def distribute_bids(self, price, total_to_distribute, step=1.02):
 
         # 4. Store the remainder of total_to_distribute, as well as the final
         #    order placed in decaying_bid_ladder
+        # TODO
```

Comment:
does this need to be done for this feature?

Reply:
No, it's being handled by the decaying ladder itself.